### PR TITLE
Added stats to the Client

### PR DESCRIPTION
**Pull request by NoodleSoup**
Python version missed the client.stats() functionality, this resolves that.

### CODE REVIEW

File: spamwatch/client.py [hunk 1]
```diff
@@ -163,3 +163,13 @@ def delete_ban(self, user_id: int) -> None:
         """Remove a ban"""
         self._make_request(f'banlist/{user_id}', method='delete')
     # endregion
+    
+    #region Stats
+    def stats(self) -> Dict[str, int]:
+        """Get ban stats"""
+        try:
```
Comment (SitiSchu) [inline spamwatch/client.py]: You can remove the entire try except as the only error this route could cause would be a 500 which is handled in _make_request

File: spamwatch/client.py [hunk 2]
```diff
@@ -163,3 +163,13 @@ def delete_ban(self, user_id: int) -> None:
         """Remove a ban"""
         self._make_request(f'banlist/{user_id}', method='delete')
     # endregion
+    
+    #region Stats
```
Comment (SitiSchu) [inline spamwatch/client.py]: this needs a space after the # to conform with PEP 8 and so its a valid region for PyCharm, same for the endregion comment

**Review by SitiSchu — CHANGES_REQUESTED** [review]
Thanks! Just two small changes and its good to be merged

**Comment by NoodleSoup** [comment]
> Thanks! Just two small changes and its good to be merged

Just added the two changes.

**Review by SitiSchu — APPROVED** [review]

**Pull request merged**